src/libcpu/src/jit/jit.cpp

```diff
@@ -65,19 +74,39 @@ initStubs()
    auto introLabel = a.newLabel();
    auto extroLabel = a.newLabel();
    auto exitLabel = a.newLabel();
+   auto verifyPreLabel = a.newLabel();
+   auto verifyPostLabel = a.newLabel();
 
    // This is invoked to set up the neccessary state for
    //  the JIT block we want to execute.
    a.bind(introLabel);
    a.push(asmjit::x86::rbp);
    a.push(asmjit::x86::rbx);
-   a.push(asmjit::x86::rdi);
+#ifdef PLATFORM_WINDOWS
```

Comment:
We should not need to depend on an ifdef for save/load here.  Just use the a.stateReg, a.membaseReg variables, along with the pool listing available at like... a.regs or something.